templates/build_script_python.sh

```diff
@@ -21,77 +21,96 @@
 PACKAGE_NAME=
 PACKAGE_VERSION=
 PACKAGE_URL=
+PACKAGE_DIR="$(pwd)/$PACKAGE_NAME"
 
-yum install -y git python3 python3-devel.ppc64le gcc gcc-c++ make wget sudo
-pip3 install pytest tox
+yum install -y git  python3 python3-devel.ppc64le gcc gcc-c++ make wget sudo
+pip3 install pytest tox nox
 PATH=$PATH:/usr/local/bin/
 
```

Comment:
Any specific reason to remove rust installation?

Reply:
No specific reason. We dont need it. We are going to improve the script going forward.